docs/user/openstack/customization.md

```diff
@@ -1,29 +1,55 @@
 # OpenStack Platform Customization
 
-In the OpenShift Installer `install-config.yaml` you can set the following options regarding the OpenStack platform:
+Beyond the [platform-agnostic `install-config.yaml` properties](../customization.md#platform-customization), the installer supports additional, OpenStack-specific properties.
 
-- `machines.platform.openstack.region` - The OpenStack region where the cluster will get created
-- `machines.platform.openstack.cloud` - Name of the OpenStack cloud to use from clouds.yaml
-- `machines.platform.openstack.externalNetwork` - The OpenStack external network name to be used for installation
-- `machines.platform.openstack.computeFlavor` - The OpenStack compute flavor to use for master servers
-- `machines.platform.openstack.lbFloatingIP` - Existing Floating IP to associate with API loadbalancer
-- `machines.platform.openstack.trunkSupport` - Whether OpenStack ports can be trunked. True or False
-- `machines.platform.openstack.octaviaSupport` - Whether OpenStack supports Octavia. True of False
-- `machines.platform.openstack.defaultMachinePlatform` - (optional) The default configuration used when installing on OpenStack for machine pools
+## Cluster-scoped properties
 
-For more technical definitions, see the [go docs](https://godoc.org/github.com/openshift/installer/pkg/types/openstack#Platform).
+* `cloud` (required string): The name of the OpenStack cloud to use from `clouds.yaml`.
+* `computeFlavor` (required? string): The OpenStack compute flavor to use for control-plane machines (FIXME: how is this different from `controlPlane.platform.openstack.type`?).
```

Comment:
This may be a regression... Looks like maybe we should conform to the standard of using controlPlane.platform.<platform>.type

Reply:
In its current state, it looks like `computeFlavor` is a required field

Reply:
yes, we use computeFlavor now and it is a required field, but it has a lower priority over platform.<platform>.type
unfortunately we can't remove it in this cycle, so I suggest to
1. make it optional in 4.2 (requires a patch for the installer),
2. deprecate it.

And then we will remove computeFlavor in 4.3

Reply:
> yes, we use computeFlavor now and it is a required field, but it has a lower priority over platform..type

I'm confused.  If you set `controlPlane.platform.openstack.type` that masks `platform.openstack.computeFlavor`, but we still error out if `platform.openstack.computeFlavor` is unset?

Reply:
Anyhow, attempted to summarize with ea6f75edd -> 715a847b0.  Let me know if I misunderstood.